### ISSUE TITLE
0.4.3.88 Cannot add second PDH counter to INI

### ISSUE THREAD

**Issue by MilanDadok** [mickem/nscp#86]
example config
[/settings/system/windows/counters/LogicalDiskCDiskTime]
type = double
counter = \LogicalDisk(C:)\% Disk Time
collection strategy = round robin
buffer size = 60s

[/settings/system/windows/counters/LogicalDiskDDiskTime]
type = double
counter = \LogicalDisk(D:)\% Disk Time
buffer size = 60s
collection strategy = round robin

nscp test output
```
E  w32system Exception in Failed to add counter LogicalDiskDDiskTime:
                    D:\source\nscp\modules\CheckSystem\pdh_thread.cpp:71
D  w32system Loading counter: LogicalDiskCDiskTime = \LogicalDisk(C:)\% Disk Time
```
there is error in not closing tmpPdh probablly
```
diff --git a/modules/CheckSystem/pdh_thread.cpp b/modules/CheckSystem/pdh_thread.cpp
index a1fa5ca..4b66475 100644
--- a/modules/CheckSystem/pdh_thread.cpp
+++ b/modules/CheckSystem/pdh_thread.cpp
@@ -67,6 +67,7 @@ void pdh_thread::thread_proc() {
 				tmpPdh.open();
 				counters_.push_back(instance);
 				lookups_[instance->get_name()] = instance;
+				tmpPdh.close();
 			} catch (const std::exception &e) {
 				NSC_LOG_ERROR_EX("Failed to add counter " + obj.alias + ": ", e);
 				continue;
```
after that change all is working normal in my test's

**Comment by mickem** [comment]
thanks!!!

pull request?

**Comment by mickem** [comment]
Should be fixed in the latest nightly build (0.4.3.90)
